src/components/SVGViewer/stories/zoom.md

```diff
@@ -0,0 +1,29 @@
+# Zoom callback
+
+<!-- STORY -->
+
+#### Description:
+
+While zooming callback is params will be fired
+
+#### Usage:
+
+```typescript jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { SVGViewer } from '@cognite/gearbox';
+
+function ExampleComponent(props) {
+
+  return (
+    <div style={{ height: '100vh' }}>
+      <SVGViewer
+        documentId={5185355395511590}
+        handleAnimateZoom={zoomCallback}
```

Comment:
Same here, better to have callback realization in the same example as described above.